rest_framework/serializers.py

```diff
@@ -562,7 +562,7 @@ def to_representation(self, data):
         """
         # Dealing with nested relationships, data can be a Manager,
         # so, first get a queryset from the Manager if needed
-        iterable = data.all() if isinstance(data, models.Manager) else data
+        iterable = data.all() if hasattr(data, 'all') else data
         return [
```

Comment:
I'm not sure this is the proper fix.
I wonder if we shouldn't simply replace `models.Manager` by `models.QuerySet`